Code/GraphMol/Descriptors/testEEM.cpp

```diff
@@ -0,0 +1,196 @@
+//  Created by Guillaume GODIN
```

Comment:
I haven't reviewed down here. I will look at this after we get the first set of changes in.